tests/drivers/dac/dac_api/src/test_dac.c

```diff
@@ -9,7 +9,13 @@
 #include <zephyr/kernel.h>
 #include <zephyr/ztest.h>
 
-#if defined(CONFIG_BOARD_NUCLEO_F091RC) || \
+#if defined(CONFIG_BOARD_SAME54_XPRO)
+
+#define DAC_DEVICE_NODE		DT_NODELABEL(dac0)
+#define DAC_RESOLUTION		12
+#define DAC_CHANNEL_ID		0
```

Comment:
Please use the section below with the same configuration (starting with `CONFIG_BOARD_TWR_KE18F`) instead of creating a new one.